main.py

```diff
@@ -2,6 +2,43 @@
 
 from django.db.models import QuerySet
 
+from db.models import Genre, Actor
+
 
 def main() -> QuerySet:
-    pass
+    genres = [
+        "Western",
+        "Action",
+        "Dramma"
```

Comment:
The genre 'Dramma' should be corrected to 'Drama' before creating the object, or ensure the update operation is intended to correct this typo.